src/keras_format/activation_config.ts

```diff
@@ -8,6 +8,38 @@
  * =============================================================================
  */
 
-/** @docinline */
-export type ActivationIdentifier = 'elu'|'hardSigmoid'|'linear'|'relu'|'relu6'|
-    'selu'|'sigmoid'|'softmax'|'softplus'|'softsign'|'tanh'|string;
+import {stringDictToArray} from './utils';
+
+/**
+ * List of all known activation names, along with a string description.
+ *
+ * Representing this as a class allows both type-checking using the keys and
+ * generating an appropriate options array for use in select fields.
```

Comment:
This feels a little odd in the spec/keras_format.  We're coupling a particular apps's usage of fields into the spec?